.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
Whats this for?

Reply:
To be honest - I'm not quite sure.
"root: special property that should be specified at the top of the file outside of any sections. Set to "true" to stop .editorconfig files search on current file." - from the website https://editorconfig.org/, but I don't quite understand what that means. I'm a decent SQF scripter, but that's where my competences end.

I think I must have imported this from the ACE3 repository some time ago, made some changes and have used it since in all of my Arma related projects.
https://github.com/acemod/ACE3/blob/master/.editorconfig - It's slightly different, but it has `root = true` too.

Reply:
As I understand it it prevents editors from going up in the directory structure for more `.editorconfig` files. 

So, like… imagine 
* you've got `./dev/.editorconfig` that configures your editor for all your projects, and then 
* you've got `./dev/ace3/.editorconfig` which overrides some of those settings

Setting `root=true` in `./dev/ace3/.editorconfig` forces the ace project to ignore your `./dev/.editorconfig`

Reply:
So in other words this has nothing to do here.

Reply:
Well, I meant the whole file not just the line

Reply:
I would strongly disagree: It allows anyone who uses the extension to get consistent tab indentations in a project, which to me quite nice/important.

I use Atom to script Arma stuff, but this extension can be used in numerous editors and IDEs to keep consistency in some settings.
It prevents new files being made from having weird or inconsistent tab styles (like [here](https://github.com/gruppe-adler/grad_trenches/blob/master/addons/functions/functions/fnc_resetDecay.sqf) or [here](https://github.com/gruppe-adler/grad_trenches/blob/master/addons/functions/functions/fnc_decayPFH.sqf)).

The only major downside I see to it is that people need to install it/run it for it to work - but that's it.
At worst this file does nothing for the user, at best it helps them when editing.